imgui/src/imgui/base64.c

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
We have functions for b64 encode/decode in the Defold SDK: 

https://defold.com/ref/stable/dmCrypt/#dmCrypt::Base64Encode:src-src_len-dst-dst_len[in,out]

We might as well use those?

Reply:
Ahh, soz. Yes we should. Will make a change and test.

Reply:
Have added dmCrypt Base64Decode